include/vigra/blockwise_labeling.hxx

```diff
@@ -88,7 +92,8 @@ blockwiseLabeling(DataBlocksIterator data_blocks_begin, DataBlocksIterator data_
                   LabelBlocksIterator label_blocks_begin, LabelBlocksIterator label_blocks_end,
                   NeighborhoodType neighborhood, Equal equal,
                   const Value* background_value,
-                  Mapping& mapping)
+                  Mapping& mapping,
+                  const int nThreads = ParallelOptions::Auto)
```

Comment:
Is there a way to set the number of threads in the highlevel API of `labelMultiArrayBlockwise()`?